Webpack not found

Application hangs after login with admin account.

### Expected results

The application is up and running

### Actual results

The application stuck right after login:

<img width="594" alt="Screenshot 2020-02-04 at 18 06 09" src="https://user-images.githubusercontent.com/304929/73768290-0a70c900-4779-11ea-862f-e3dcb2b9bcce.png">

#### How to reproduce the bug
0. git clone
1. docker-compose up
2. open http://localhost:8088/
3. login with admin/admin


### Environment

(please complete the following information):

- superset version: current master 540df268c286d454ad3980f5c38a972d2163344f
- python version: `python --version`
- node.js version: `node -v`
- npm version: `npm -v`

### Checklist

Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

```
superset-node_1    | audited 906857 packages in 74.627s
superset-node_1    |
superset-node_1    | 1 package is looking for funding
superset-node_1    |   run `npm fund` for details
superset-node_1    |
superset-node_1    | found 2 moderate severity vulnerabilities
superset-node_1    |   run `npm audit fix` to fix them, or `npm audit` for details
superset-node_1    |
superset-node_1    | > superset@0.999.0-dev dev /app/superset/assets
superset-node_1    | > webpack --mode=development --colors --progress --debug --watch
superset-node_1    |
superset-node_1    | sh: 1: webpack: not found
superset-node_1    | npm ERR! code ELIFECYCLE
superset-node_1    | npm ERR! syscall spawn
superset-node_1    | npm ERR! file sh
superset-node_1    | npm ERR! errno ENOENT
superset-node_1    | npm ERR! superset@0.999.0-dev dev: `webpack --mode=development --colors --progress --debug --watch`
superset-node_1    | npm ERR! spawn ENOENT
superset-node_1    | npm ERR!
superset-node_1    | npm ERR! Failed at the superset@0.999.0-dev dev script.
superset-node_1    | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -69,7 +69,7 @@ services:
 
   superset-node:
     image: node:10-jessie
-    command: ["bash", "-c", "cd /app/superset-frontend && npm install && npm run dev"]
+    command: ["bash", "-c", "cd /app/superset-frontend && npm install --global webpack webpack-cli && npm install && npm run dev"]
     env_file: docker/.env
     depends_on: *superset-depends-on
     volumes: *superset-volumes

